compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LibraryBuilder.cpp

```diff
@@ -491,6 +516,33 @@ LibraryBuilder::buildLibraryV0ExportTable(std::string libraryName) {
         exportSrcLocsType, global, ArrayRef<llvm::Constant *>{zero, zero});
   }
 
+  // iree_hal_executable_src_file_v0_t::src_files;
+  llvm::Constant *exportSrcFiles =
+      llvm::Constant::getNullValue(srcLocType->getPointerTo());
+  if (mode == Mode::INCLUDE_REFLECTION_ATTRS) {
+    SmallVector<llvm::Constant *> exportSrcFileValues;
+    for (auto dispatch : exports) {
```

Comment:
this granularity doesn't work - we don't want a source file per export as all exports may come from the same file or any combination of files